<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestaocerveja</class>
 <widget class="QDialog" name="fm_gestaocerveja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cerveja</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>691</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Adicionar Cerveja</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>282</width>
       <height>182</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Nome</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_add_nome"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Validade</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_add_lote"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Estilo</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_add_estilo"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_add_salvar">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
          </property>
          <property name="text">
           <string>Salvar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_add_limpar">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
          </property>
          <property name="text">
           <string>Limpar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Gest√£o Cerveja</string>
    </attribute>
    <widget class="QTableWidget" name="tw_gc_listar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>281</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>212</width>
       <height>82</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Filtar por:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <item>
           <widget class="QRadioButton" name="rb_gc_idCerveja">
            <property name="text">
             <string>ID Cerveja</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_gc_nome">
            <property name="text">
             <string>Nome</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Filtro</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_gc_filtrar"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_gv_filtrar">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
          </property>
          <property name="text">
           <string>Filtrar</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>170</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_17">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_gc_nome"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Validade</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_gc_lote"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Estilo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_gc_estilo"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButtonSalvar_2">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
          </property>
          <property name="text">
           <string>Salvar</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonExcluir_2">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
          </property>
          <property name="text">
           <string>Excluir</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
